sky/backends/cloud_vm_ray_backend.py

```diff
@@ -3547,6 +3547,23 @@ def post_teardown_cleanup(self,
                             cluster_name=handle.cluster_name,
                             stdout=tpu_stdout,
                             stderr=tpu_stderr))
+        if handle.launched_resources.is_image_managed and terminate:
```

Comment:
Q: say I clone `old` into `new`. Then I run `sky stop new` and `sky launch -c new <some other resource args>`. Does it override its `handle.launched_resources.is_image_managed`?